main.go

```diff
@@ -356,13 +368,21 @@ func main() {
 	archiveCmd.SetDisableIndexWhileBuilding(cfg.DisableIndexWhileBuilding)
 	archiveCmd.SetArchivePath(tmpArchivePath)
 
+	destination := "generic/platform=" + string(platform)
+	options := []string{"-destination", destination}
 	if cfg.XcodebuildOptions != "" {
-		options, err := shellquote.Split(cfg.XcodebuildOptions)
+		userOptions, err := shellquote.Split(cfg.XcodebuildOptions)
 		if err != nil {
 			fail("Failed to shell split XcodebuildOptions (%s), error: %s", cfg.XcodebuildOptions)
 		}
-		archiveCmd.SetCustomOptions(options)
+
+		if sliceutil.IsStringInSlice("-destination", userOptions) {
```

Comment:
Question: this is the only way a -destination can be set? For example is `"-destination=generic/platform=ios"`

Not sure if worth mentioning this...

Reply:
`xcodebuild -showBuildSettings -destination=""`

```
User defaults from command line:
    destination =
```

`xcodebuild -showBuildSettings -destination ""`

```
xcodebuild: error: option 'Destination' requires at least one parameter of the form 'key=value'
```

So the form `-flag=value` treated differently than `-flag value`.